_posts/fr/newsletters/2022-10-05-newsletter.md

```diff
@@ -91,8 +91,8 @@ d'infrastructure Bitcoin.
     [news219 ratecards]) pourraient plutôt devenir des
     *maximum amount per-forward ratecards*, où celui qui effectue la dépense se verrait facturer
     un feerate plus bas pour envoyer de petits paiements et un feerate plus élevé
-    pour envoyer des paiements plus importants.  A la différence de la proposition
-    originale de ratecards, il s'agirait de montants absolus et non relatifs au
+    pour envoyer des paiements plus importants. A la différence de la proposition
+    originale de carte de taux, il s'agirait de montants absolus et non relatifs au
```

Comment:
Faut-il traduire l'idée présentée de " fee ratecards" en "carte de taux de frais" pour la compréhension générale ?
c'est la raison pour laquelle j'avais laissé l'original.

Reply:
in my opinion, we can keep "fee ratecard" in quotes/title like above these lines, and take the French translation otherwise. WDYT?

Reply:
I agree

Reply:
@Copinmalin @acidbunny21 Is this comment resolved? Anything else to address for the PR?

Reply:
@bitschmidty IT's OK you can merge this PR